src/test/java/com/google/edith/UserInsightsTest.java

```diff
@@ -153,66 +160,100 @@ public void createJson_returnsCorrectJsonString() {
     List<Item> itemProperties = new ArrayList<>();
 
     Entity newEntity = new Entity(items.get(0));
-    setEntityProperties(newEntity, "corn", 5, 1, "2020-06-29");
-    itemProperties.add(new Item("corn", 5.00, 1L, "2020-06-29"));
+    setEntityProperties(newEntity, "corn", USER_ID, "vegetable", 5, 1, "2020-06-29", RECEIPT_ID);
+    itemProperties.add(Item.builder()
+                           .setName("corn")
+                           .setUserId(USER_ID)
+                           .setCategory("vegetable")
+                           .setPrice(5)
+                           .setQuantity(1)
+                           .setDate("2020-06-29")
+                           .setReceiptId(RECEIPT_ID)
+                           .build());
+    // itemProperties.add(new Item("corn", USER_ID, "vegetable", 5.00, 1L, "2020-06-29", RECEIPT_ID));
```

Comment:
can these commented lines be removed as well

Reply:
Done.